glue/qt/widget_properties.py

```diff
@@ -237,7 +237,7 @@ def connect_current_combo(client, prop, widget):
     """
 
     def _push_combo(value):
-        idx = widget.findData(value)
+        idx = _find_combo_data(widget, value)
```

Comment:
As indicated in #755, I think we actually need to merge this first, so I'm actually just going to merge in the changes from 771 here to make things easier.